styles/scss/core/bootstrap-override/mixins/_grid-framework.scss

```diff
@@ -0,0 +1,80 @@
+// Framework grid generation
+//
+// Used only by Bootstrap to generate the correct number of grid classes given
+// any value of `$grid-columns`.
+
+@mixin make-grid-columns($columns: $grid-columns, $gutter: $grid-gutter-width, $breakpoints: $grid-breakpoints) {
+  // Common properties for all breakpoints
+  %grid-column {
+    position: relative;
+    width: 100%;
+    padding-right: calc($gutter * .5);
+    padding-left: calc($gutter * .5);
```

Comment:
In these files, we have small changes

For example `padding-right: $gutter * .5` to `padding-right: calc($gutter * .5)`